application-xpoll-ui/src/main/resources/XPoll/XPollJSExtension.xml

```diff
@@ -147,7 +147,11 @@
           $('#xpollSaveForm').load(XWiki.currentDocument.getURL('get') +  ' #xpollSaveForm&gt;*');
         },
         error: function(data) {
-          new XWiki.widgets.Notification("$services.localization.render('contrib.xpoll.vote.failed', 'HTTP Error')", 'error');
+          if (data.status == 403) {
+            new XWiki.widgets.Notification("$services.localization.render('contrib.xpoll.vote.failed.forbidden')", 'error');
+          } else {
+            new XWiki.widgets.Notification("$services.localization.render('contrib.xpoll.vote.failed', 'HTTP Error')", 'error');
```

Comment:
I just noticed that we create a new notification instead of replacing ``progressNotification``. Can you:

* drop the ``complete`` callback (not needed)
* use ``this.progressNotification.replace(...)`` in both ``success`` and ``error`` callbacks.